### PR TITLE
updating response documentation

### PR DESCRIPTION
**Pull request by Courey**

### CODE REVIEW

File: api/external/response.html [hunk 1]
```diff
@@ -308,9 +397,9 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
       "last_name": "Smith",
       "favorite_food": "tacos"
     },
-    "timestamp": "@D:2011-01-02T10:27:43",
+    "timestamp": "@D:2018-01-12T10:27:4",
```
Comment (JoshMock) [inline api/external/response.html]: shouldn't the seconds value on this always be two digits? if so there's a couple examples here that need to be tweaked slightly.

File: api/external/response.html [hunk 2]
```diff
@@ -555,6 +677,15 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
 </p>
 </dd></dl>
 
+<dl class="get">
+<dt id="get--#account_id-response-#mailing_id-opens">
+<tt class="descname">GET </tt><tt class="descname">/#account_id/response/#mailing_id/opens/#member_id</tt><a class="headerlink" href="#get--#account_id-response-#mailing_id-opens-#member_id" title="Permalink to this definition">¶</a></dt>
```
Comment (JoshMock) [inline api/external/response.html]: this `href` doesn't match with the `id` in the `<dt>` tag a couple lines up.

File: api/external/response.html [hunk 3]
```diff
@@ -678,6 +816,54 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
 </p>
 </dd></dl>
 
+<dl class="get">
+<dt id="get--#account_id-response-#mailing_id-clicks">
+<tt class="descname">GET </tt><tt class="descname">/#account_id/response/#mailing_id/clicks/#member_id</tt><a class="headerlink" href="#get--#account_id-response-#mailing_id-clicks-#member_id" title="Permalink to this definition">¶</a></dt>
```
Comment (JoshMock) [inline api/external/response.html]: these ones don't match either

File: api/external/response.html [hunk 4]
```diff
@@ -938,13 +1253,50 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
...
+mailing type - &#8216;m&#8217; for standard mailings, &#8216;t&#8217; for test mailings, &#8216;s&#8217; for subject split test mailings, and
 &#8216;r&#8217; for trigger mailings.</td>
 </tr>
 </tbody>
 </table>
 </dd></dl>
 
+<dl class="put">
+<dt id="get--#account_id-response-#share_id-customer_share">
+<tt class="descname">PUT </tt><tt class="descname">/#account_id/response/#mailing_id/customer_share_status</tt><a class="headerlink" href="#put--#account_id-response-#share_id-customer_share_status" title="Permalink to this definition">¶</a></dt>
```
Comment (JoshMock) [inline api/external/response.html]: here too

File: api/external/response.html [hunk 5]
```diff
@@ -969,8 +1321,146 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
...
+
+</pre>
+</div>
+</p>
+</dd>
+</dl>
+
+<dl class="get">
+<dt id="get--#account_id-response">
+<tt class="descname">GET </tt><tt class="descname">/#account_id/response/#mailing_id/split_test_info</tt><a class="headerlink" href="#get--#account_id-response-#mailing_id-split_test_info" title="Permalink to this definition">¶</a></dt>
```
Comment (JoshMock) [inline api/external/response.html]: and here

File: api/external/response.html [hunk 6]
```diff
@@ -555,6 +677,15 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
 </p>
 </dd></dl>
 
+<dl class="get">
+<dt id="#get--#account_id-response-#mailing_id-opens-#member_id">
```
Comment (lroggendorff) [inline api/external/response.html]: 👋 Hi-dilly-o! The `id` attribute can lose the leading `#` on this here line.

File: api/external/response.html [hunk 7]
```diff
@@ -938,13 +1253,50 @@ <h1>Response<a class="headerlink" href="#response" title="Permalink to this head
...
-mailing type - &#8216;m&#8217; for standard mailings, &#8216;t&#8217; for test mailings and
+mailing type - &#8216;m&#8217; for standard mailings, &#8216;t&#8217; for test mailings, &#8216;s&#8217; for subject split test mailings, and
 &#8216;r&#8217; for trigger mailings.</td>
 </tr>
 </tbody>
 </table>
 </dd></dl>
 
+<dl class="put">
+<dt id="get--#account_id-response-#share_id-customer_share_status">
```
Comment (lroggendorff) [inline api/external/response.html]: Looks like this one needs `put` as the verb at the beginning of the `id` here.

**Pull request merged**